sal_interfaces/LinearStage/LinearStage_Events.xml

```diff
@@ -28,7 +28,7 @@
             <EFDB_Name>heartbeat</EFDB_Name>
             <Description>Beats the drum.</Description>
```

Comment:
?

Reply:
Copy and paste from Russell's heartbeat event.